app/src/main/java/com/yacgroup/yacguide/AboutActivity.kt

```diff
@@ -55,4 +56,11 @@ class AboutActivity : BaseNavigationActivity() {
         val intent = Intent(this, PrivacyPolicyActivity::class.java)
         startActivity(intent)
     }
+
+    @Suppress("UNUSED_PARAMETER")
+    fun showLicense(v: View) {
+        val openURL = Intent(android.content.Intent.ACTION_VIEW)
+        openURL.data = Uri.parse(getString(R.string.license_url))
```

Comment:
Do we need error handling here? E.g. if there is no browser available?

Reply:
Can this happen? An Android without browser? I will check.

Reply:
The other approach would be, compiling the license text into the app. Which has the advantage, that it can be shown even without internet connection.

Reply:
No no. Opening a browser is a good solution here. I just don't know what would happen if there is no browser available.